### DISCUSSION TITLE
browsersync

### DISCUSSION THREAD

**Discussion by benthien** [foundation/proton#21] [General]
In my web sites I use some php along with html. I use a local server to preview changes. Previously I set a proxy to a virtual host on my local server in the gulp setup, i.e., I set `proxy: "http://<virtual host name>"` under `function server` in gulpfile.babel.js . Is there anything similar in proton?

**Comment by joeworkman** [comment]
Sorry for the delay. You should be able to point any web server to the dist folder to preview the site.
